extensions/amp-accordion/1.0/accordion.js

```diff
@@ -426,7 +475,9 @@ export function AccordionContent({
     <Comp
       {...rest}
       ref={ref}
-      className={`${className} ${classes.sectionChild} ${classes.content}`}
+      className={`${className} ${classes.sectionChild} ${classes.content} ${
+        !expanded && useDisplayLocking ? classes.contentHiddenMatchable : ''
+      }`}
       id={contentId}
```

Comment:
If the display locking feature is enabled and the browser supports the necessary CSS property.  Then add the CSS classes needed for display locking feature.